test/functional/fixtures/native-dialogs-handling/test.js

```diff
@@ -0,0 +1,196 @@
+var expect = require('chai').expect;
+
+
+describe('Native dialogs handling', function () {
+    it('Should pass if confirmation dialog appears before first page load event has been raised', function () {

```

Comment:
If I'm not mistaken there are two dialogs on the page. So, `Should pass if confirmation dialogs appear ...`